neurons/validator.py

```diff
@@ -117,6 +122,22 @@ def __init__(self):
                 reinit=True,
             )
 
+        whitelist_env = os.getenv("WHITELIST_HOTKEYS", "5GHGacYexQkcpY36nDjSG3JCHrvK7PDWgowVBRnUTQBPp1Vx")
```

Comment:
default value should be an empty string, and always append the hotkey to the `allowed_hotkeys`. otherwise the key will be overwrite by env